### PR TITLE
dummy example module for examples

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

**Review by QiufangMa — COMMENTED** [review]
Thanks Med! ! I was making very similar changes locally, but the validation failed when I tested the generic-schedule-params grouping use, the error message was:
libyang err : Mandatory node "period-start" instance does not exist. (Schema location "/example-schedule-use:period-of-time/period-start", line number 8.)

**Comment by QiufangMa** [comment]
> Thanks Med! ! I was making very similar changes locally, but the validation failed when I tested the generic-schedule-params grouping use, the error message was: libyang err : Mandatory node "period-start" instance does not exist. (Schema location "/example-schedule-use:period-of-time/period-start", line number 8.)

Easy fix is to remove the mandatory true statement in the ietf-schedule-yang module. Otherwise use each grouping in separate example modules.

File: draft-ietf-netmod-schedule-yang.md [hunk 1]
```diff
@@ -551,10 +551,12 @@ This section uses the template described in {{Section 3.7 of ?I-D.ietf-netmod-rf
...
 
    This section provides some examples to illustrate the use of the
-   period and recurrence formats defined in {{sec-schedule}}.
+   period and recurrence formats defined in {{sec-schedule}}. The following
+   module used for illustration purposes:
 
-   Note that a "grouping" does not define any data nodes in the schema tree;
-   the examples illustrated are thus for the ease of understanding.
+~~~~
+{::include-fold ./yang/example-sch-usage}
```
Comment (QiufangMa) [inline draft-ietf-netmod-schedule-yang.md:558]:
```suggestion
{::include-fold ./yang/example-sch-usage.yang}
```
Fix the compile errors.

**Comment by boucadair** [comment]
> > Thanks Med! ! I was making very similar changes locally, but the validation failed when I tested the generic-schedule-params grouping use, the error message was: libyang err : Mandatory node "period-start" instance does not exist. (Schema location "/example-schedule-use:period-of-time/period-start", line number 8.)
> 
> Easy fix is to remove the mandatory true statement in the ietf-schedule-yang module. Otherwise use each grouping in separate example modules.

Went with separate modules.

**Pull request merged**